compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposePanel.desktop.kt

```diff
@@ -46,8 +47,29 @@ import org.jetbrains.skiko.SkiaLayerAnalytics
 class ComposePanel @ExperimentalComposeUiApi constructor(
     private val skiaLayerAnalytics: SkiaLayerAnalytics,
 ) : JLayeredPane() {
+    private var reducePresentationLatency: Boolean? = null
```

Comment:
Why not just `vsyncEnabled` to have 1-1 match with skiko property that it's actually set?

Reply:
Because it's a bit misleading. We still perform frame-rate(vsync-based) throttling (at least on macOS).
This change in fact affects only macOS presentation configuration (which is exclusive to macOS).